consumer/src/main/java/com/flipkart/varadhi/consumer/PolledMessageTracker.java

```diff
@@ -20,22 +19,11 @@ public PolledMessageTracker(Consumer<O> committer, PolledMessage<O> message) {
 
     @Override
     public Message getMessage() {
-        return new PolledMessageWrapper<>(message);
+        return message;
     }
 
     @Override
     public void onConsumed(MessageConsumptionStatus status) {
         committer.commitIndividualAsync(message);
     }
-
```

Comment:
since PolledMessage is now a Message, we don't need to implement a Wrapper to make it work with the MessageTracker contracts